catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesLuceneIndexer.java

```diff
@@ -152,6 +152,14 @@ public void updateProgress(final int progress) {
     }
```

Comment:
Should this class be renamed?

Reply:
Curious as to why?  The behavior has not changed